Fix inline vs. non-inline namespace in GCS+gRPC

The `.proto` file uses the `v1` namespace but proto does not use inline namespaces:

```
bazel-out/k8-fastbuild/bin/google/cloud/storage/internal/grpc_resumable_upload_session_url.pb.h:60:11: warning: inline namespace reopened as a non-inline namespace
namespace v1 {
          ^
inline 
./google/cloud/storage/internal/grpc_resumable_upload_session_url.h:24:18: note: previous definition is here
inline namespace STORAGE_CLIENT_NS {
                 ^
./google/cloud/storage/version.h:23:3: note: expanded from macro 'STORAGE_CLIENT_NS'
  GOOGLE_CLOUD_CPP_VEVAL(STORAGE_CLIENT_VERSION_MAJOR, \
  ^
./google/cloud/version.h:23:40: note: expanded from macro 'GOOGLE_CLOUD_CPP_VEVAL'
#define GOOGLE_CLOUD_CPP_VEVAL(Ma, Mi) GOOGLE_CLOUD_CPP_VCONCAT(Ma, Mi)
                                       ^
./google/cloud/version.h:22:42: note: expanded from macro 'GOOGLE_CLOUD_CPP_VCONCAT'
#define GOOGLE_CLOUD_CPP_VCONCAT(Ma, Mi) v##Ma
                                         ^
<scratch space>:139:1: note: expanded from here
v1
^
```

--- a/google/cloud/storage/internal/grpc_resumable_upload_session_url.cc
+++ b/google/cloud/storage/internal/grpc_resumable_upload_session_url.cc
@@ -24,6 +24,8 @@ namespace storage {
 inline namespace STORAGE_CLIENT_NS {
 namespace internal {
 
+using ::google::storage::v1::internal::GrpcResumableUploadSessionUrl;
+
 auto constexpr kUriScheme = "grpc://";
 
 std::string EncodeGrpcResumableUploadSessionUrl(

--- a/google/cloud/storage/internal/grpc_resumable_upload_session_url.proto
+++ b/google/cloud/storage/internal/grpc_resumable_upload_session_url.proto
@@ -14,7 +14,7 @@
 
 syntax = "proto3";
 
-package google.cloud.storage.v1.internal;
+package google.storage.v1.internal;
 
 message GrpcResumableUploadSessionUrl {
   string bucket_name = 1;

